### PR TITLE
new pynos call to enable/disable fabric neighbor on a interface

### PR DESCRIPTION
**Pull request by uday83**
Leaf1-2# show run int t 1/0/15
interface TenGigabitEthernet 1/0/15
 channel-group 200 mode on type standard
 no fabric isl enable
 no fabric trunk enable
 fabric neighbor-discovery disable
 no shutdown

to disable/enable  fabric neighbor-discovery disable on a interface

### CODE REVIEW

File: pynos/versions/base/interface.py [hunk 1]
```diff
@@ -4001,3 +4001,81 @@ def ipv6_link_local(self, **kwargs):
...
+            ...         # doctest: +IGNORE_EXCEPTION_DETAIL
+            Traceback (most recent call last):
+            KeyError
+        """
+        int_type = str(kwargs.pop('int_type').lower())
+        name = str(kwargs.pop('name'))
+        enabled = kwargs.pop('enabled', True)
+        callback = kwargs.pop('callback', self._callback)
+
+        int_types = [
```
Comment (samirsss) [inline pynos/versions/base/interface.py:4051]: is this not supported on gigabitethernet?

**Review by samirsss — CHANGES_REQUESTED** [review]

File: pynos/versions/base/interface.py (hunk 1 above)
Comment (uday83) [reply pynos/versions/base/interface.py:4051]: fabric commands on 1G are not supported. VCS formation happens only on links with 10G and above speeds .

**Comment by samirsss** [comment]
Feel free to merge this.

**Review by samirsss — APPROVED** [review]

**Comment by samirsss** [comment]
I am going to merge this for you since I need to rev up the version of pynos today.

**Pull request merged**